client/lib/importer/actions.js

```diff
@@ -10,6 +10,14 @@ const wpcom = require( 'lib/wp' ).undocumented();
 import { actionTypes } from './constants';
 import { fromApi, toApi } from './common';
 
+const apiSuccess = (...args) => {
+	Dispatcher.handleViewAction( {
+		type: actionTypes.API_SUCCESS
+	} );
+
+	return args;
```

Comment:
Good point. The promise chain should only ever send a single argument down the list; I was trying to make sure I didn't mangle the arguments but ended up doing just that. I have changed this in 843e62c89f29830477508d8b80469cb53d3e39da to treat the input and output as a single variable.